charts/rancher-istio/rancher-istio/100.0.1+up1.10.4/templates/istio-install-job.yaml

```diff
@@ -29,8 +29,6 @@ spec:
             value: {{ .Values.forceInstall | default "false" | quote }}
           - name: RELEASE_MIRROR_ENABLED
             value: {{ .Values.installer.releaseMirror.enabled | quote }}
-          - name: DEBUG
```

Comment:
Does Istio no longer support the debug flag? Or does it assume debug if secondSleep > 0 now?

Reply:
It assumes debug if secondsSleep > 0

Reply:
Gotcha. I see the change in https://github.com/rancher/istio-installer/commit/08a77173f6e13654490de3896425fb2a163ea8ac.